src/control/tools/EditSelection.cpp

```diff
@@ -731,20 +688,20 @@ auto EditSelection::getSelectionTypeForPos(double x, double y, double zoom) -> C
         return CURSOR_SELECTION_BOTTOM_RIGHT;
     }
 
-    if (x1 - (20 + this->btnWidth) - BORDER_PADDING <= x && x1 - (20 + this->btnWidth) + BORDER_PADDING >= x &&
-        y1 - BORDER_PADDING <= y && y1 + BORDER_PADDING >= y) {
+    if (xmin - (20 + this->btnWidth) - BORDER_PADDING <= x && x <= xmin - (20 + this->btnWidth) + BORDER_PADDING &&
```

Comment:
Could you please define `constexpr` for these magical numbers 20 and 8 which are used (here and elsewhere) for the position of the delete and rotate rectangles? That would make it more readable.